internal/image/resource_image_test.go

```diff
@@ -572,3 +660,82 @@ resource "incus_image" "img1" {
 }
 	`, projectName, instanceName, acctest.TestImage)
 }
+
+func testAccSourceFileSplitImage_exportImage(target string) string {
+	return fmt.Sprintf(`
+resource "incus_image" "img1" {
+  source_image = {
+    remote       = "images"
+    name         = "alpine/edge"
+    copy_aliases = true
+  }
+}
+
+resource "null_resource" "export_img1" {
+  provisioner "local-exec" {
+    command = "incus image export ${incus_image.img1.fingerprint} %s"
+  }
```

Comment:
@stgraber Here as well as in the test for the unified image I use the Terrafrom `null_resource` with `local-exec` to call `incus` and perform the necessary operations. What do you think about this approach?

Reply:
The acceptance tests pass in CI as well as on my machine: https://github.com/lxc/terraform-provider-incus/actions/runs/11931320144/job/33253958154?pr=169#step:8:141

Reply:
Yeah, that's fine